pom.xml

```diff
@@ -14,7 +14,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <groovy.version>1.8.9</groovy.version>
+    <groovy.version>2.4.7</groovy.version>
```

Comment:
That used by Jenkins 2.7.x and later LTS lines. We _could_ continue to develop for 1.x, but there are so many `Closure`-based methods added in `DefaultGroovyMethods` in 2.x that we would be missing coverage for a lot of common scripts.